src/ibrowse_lib.erl

```diff
@@ -439,4 +468,15 @@ parse_url_test() ->
               ?assertMatch(Expected_result, parse_url(Url))
       end, Urls).
 
+unquote_test() ->
+    ?assertEqual("foo bar",
+        unquote("foo+bar")),
+    ?assertEqual("foo bar",
+        unquote("foo%20bar")),
+    ?assertEqual("foo\r\n",
+        unquote("foo%0D%0A")),
+    ?assertEqual("foo\r\n",
+        unquote(<<"foo%0D%0A">>)),
```

Comment:
This looks like the same as right above.

Reply:
The difference is `binary()` vs `string()`.

Reply:
Thanks, @iilyak !

Reply:
Thanks, @iilyak @bessbd